pkg/runtime/docker/client.go

```diff
@@ -133,7 +133,7 @@ func (dc *dockerClient) AttachContainer(container string) (err error) {
 func (dc *dockerClient) RunContainer(image meta.OCIImageRef, config *runtime.ContainerConfig, name, id string) (string, error) {
 	binds := make([]string, 0, len(config.Binds))
 	for _, bind := range config.Binds {
-		binds = append(binds, fmt.Sprintf("%s:%s", bind.HostPath, bind.ContainerPath))
+		binds = append(binds, fmt.Sprintf("%s:%s", bind.HostPath, bind.ContainerPath, bind.MountOptions))
```

Comment:
Your editor is likely complaining about this line.
The format string does not match the number of arguments.

`bind.MountOptions` is a `[]string`, so you'd want to join them somehow.
This is one of the reasons it might be easier/cleaner to move to `Mounts` instead of `Binds` for docker's container spec.